mobile/src/components/common/stack/stack.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react';
+
+import { StackDirection } from '~/common/enums/enums';
+
+import { View } from '../common';
+
+type Props = {
+  children: any[];
```

Comment:
```suggestion
  children: ReactNode;
```

Reply:
I often saw typing like "ReactNode | ReactNode[]" does it fit here, or our component always receives only one node?

Reply:
ReactNode is enough for any number of children)

Reply:
Oh, thanks!

Reply:
if I set ReactNode i have lint error 
![image](https://i.gyazo.com/dbe70f4d9bdb8d0dc411a296dd5d1f6a.png)

Reply:
It's because you can't return children from component. Replace `return children;` with `return <>{children}</>;`. It will solve error

Reply:
Anyone know how fix this error?

Reply:
What about the above message?

Reply:
solved